diff --git a/dist/chunk-W3HASNBP.js b/dist/chunk-W3HASNBP.js
index 6578c9c1abe3ccb0dbfbc96e03ad85ca360289e2..a8f4749b691f7e0e390d790533f8cf6078560329 100644
--- a/dist/chunk-W3HASNBP.js
+++ b/dist/chunk-W3HASNBP.js
@@ -1 +1 @@
-import{a}from"./chunk-3EUVTL6D.js";import o from"react";import g from"clsx";import{Paginator as e,Pagination as s}from"@salt-ds/lab";var f=({className:n,pageCount:i,page:t,onPageChange:r})=>o.createElement("div",{className:g(a.root,n)},o.createElement(s,{count:i,onPageChange:r,page:t},o.createElement(e,{className:a.paginator})));export{f as a};
+import{a}from"./chunk-3EUVTL6D.js";import o from"react";import g from"clsx";import{Paginator as e,Pagination as s}from"@salt-ds/core";var f=({className:n,pageCount:i,page:t,onPageChange:r})=>o.createElement("div",{className:g(a.root,n)},o.createElement(s,{count:i,onPageChange:r,page:t},o.createElement(e,{className:a.paginator})));export{f as a};
