diff --git a/dist/index.d.ts b/dist/index.d.ts
index 68619729fe89351fb67895f8f72260981a8c3fec..25a55122b249d101c40243a5b24f23004daf2a86 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -160,7 +160,9 @@ declare function mocked<T>(item: T, options: {
 }): MaybePartiallyMockedDeep<T>;
 declare function mocked<T>(item: T): MaybeMocked<T>;
 
-type Queries = ReturnType<typeof within<typeof queries>>;
+// Prevents "'>' expected" being thrown, this is fixed in Typescript 4.7
+type typeOfWithin = typeof within;
+type Queries = ReturnType<typeOfWithin<typeof queries>>;
 declare module '@storybook/csf' {
     interface Canvas extends Queries {
     }
